astropy/wcs/setup_package.py

```diff
@@ -6,15 +6,14 @@
 import shutil
 import sys
 from collections import defaultdict
-from os.path import join
 from pathlib import Path
 
 from numpy import get_include as get_numpy_include
 from setuptools import Extension
 
 from extension_helpers import get_compiler, import_file, pkg_config, write_if_different
 
-WCSROOT = os.path.relpath(os.path.dirname(__file__))
+WCSROOT = Path(__file__).parent.resolve().relative_to(Path.cwd())
```

Comment:
I don't see how using `Path` here improves anything. `parent`, `resolve`, `cwd` are all extra steps I need to think about when reading this line.

Reply:
`.parent` = `os.path.dirname`
`relative_to(Path.cwd())` = `os.path.relpath` (which makes a relative path, but to what? It's given only 1 argument, so what it's relative to isn't obvious. Reading the docs it has a second argument, `start=os.curdir` which then makes this exactly the same as the more explicit `.relative_to(Path.cwd())`).

So as far as I can tell, the major difference is only `.resolve()`.

Reply:
I don't think `resolve()` is even needed here because `Path(__file__)` already produces an absolute path.